homeassistant/components/vesync/config_flow.py

```diff
@@ -20,11 +25,45 @@
 )
 
 
+class VesyncOptionFlowHandler(OptionsFlow):
+    """Handle an option flow for Vesync."""
+
+    def __init__(self, config_entry: ConfigEntry) -> None:
+        """Initialize option flow instance."""
+        self.config_entry = config_entry
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Manage the vesync options."""
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        options = {
+            vol.Optional(
+                CONF_SCAN_INTERVAL,
+                default=self.config_entry.options.get(
+                    CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL
+                ),
+            ): int,
```

Comment:
We do not allow configuration of the scan interval.

We should provide a sane default out of the box. In case the default isn't correct, we should adjust that.

If an user wants to use a different schedule or a conditional scan interval, they can use the homeassistant.update_entity service instead.